docs/api/utility-process.md

```diff
@@ -0,0 +1,85 @@
+# UtilityProcess
+
+`UtilityProcess` is the equivalent of [`child_process.fork`][] API from Node.js
+but instead uses [Services API][] from Chromium to create child processes with
+Node.js and Message ports enabled.
+
+`UtilityProcess` is an [EventEmitter][event-emitter].
+
+## Class: UtilityProcess
+
+> Child process with Node.js integration spawned by Chromium.
+
+Process: [Main](../glossary.md#main-process)<br />
+
+### `new UtilityProcess(modulePath[, args][, options])`
+
+* `modulePath` string - Absolute path to the script that should run as entrypoint in the child process.
```

Comment:
This should probably match https://nodejs.org/api/worker_threads.html#new-workerfilename-options for consistency!

Reply:
The api is modeled after https://nodejs.org/api/child_process.html#child_processforkmodulepath-args-options since it is closer to it in behavior.

Reply:
ah, a bit disappointing that Node diverges in those two APIs.